dsp/contact.md

```diff
@@ -0,0 +1,13 @@
+---
+title: DSP Contact
+description: Contact information for the Digital Solutions Platform (DSP) team
+---
+
+[&larr; back to Overview](/dsp)
+
+# Contact
+- [Slack channel](localhost)
+- [MS Teams channel](localhost)
+- [Email](localhost)
+- [Confluence](localhost)
+- [Jira Helpdesk](localhost)
```

Comment:
Could maybe be part of the footer with proper links. https://github.com/vgbs/dsp-documentation/blob/595617b0c412aa40538c50415350b0a2c068912b/_layouts/default.html#L32C1-L40C14